### PR TITLE
Common: run tests after successful automatic build 

### PR DESCRIPTION
**Pull request by awegrzyn**
@ktf Could you please take a look before merging?

### CODE REVIEW

File: common-o2.sh [hunk 1]
```diff
@@ -45,3 +48,7 @@ prepend-path LD_LIBRARY_PATH \$::env(COMMON_O2_ROOT)/lib
 $([[ ${ARCHITECTURE:0:3} == osx ]] && echo "prepend-path DYLD_LIBRARY_PATH \$::env(COMMON_O2_ROOT)/lib")
 EoF
 mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+
+if [[ $ALIBUILD_O2_TESTS ]]; then
+  CTEST_OUTPUT_ON_FAILURE=1 ctest ${JOBS+-j $JOBS}
+fi
```
Comment (matthiasrichter) [inline common-o2.sh]: Out of curiosity, why is it not enough to have this test in the incremental recipe?

Comment (awegrzyn) [reply common-o2.sh]: I wish I knew, I followed exactly how it's implemented in `o2.sh`

Comment (ktf) [reply common-o2.sh]: Well, in general we want to run the tests also during a regular build, e.g. when building a release. Or am I misunderstanding your question? This of course is merely policy...@brinick can you check this actually builds fine and merge it?

**Comment by awegrzyn** [comment]
@ktf @brinick @dberzano Any updates on that?

**Comment by ktf** [comment]
This actually broke the build, which will teach me to actually approve untested PRs to speed up things. :-/

@awegrzyn can you please look into it?

https://github.com/AliceO2Group/AliceO2/pull/783

Thanks.

**Comment by awegrzyn** [comment]
@PascalBoeschoten could you take a look asap? This might be connected with this issue: https://alice.its.cern.ch/jira/browse/OCCF-53

**Comment by PascalBoeschoten** [comment]
There's a fix, but some other OSX errors came to light. We're working on it..
https://github.com/AliceO2Group/Common/pull/9

**Pull request merged**